### PR TITLE
Add a focus() method to UiTextbox

### PR DESCRIPTION
**Pull request by myerscw**
Using this library while learning Vue and wanted a way to focus a ui-textbox.

If there's already a way to do this, ignore the pr!

### CODE REVIEW

File: src/UiTextbox.vue [hunk 1]
```diff
@@ -309,6 +309,11 @@ export default {
             if (this.autosizeInitialized) {
                 autosize.update(this.$refs.textarea);
             }
+        },
+
+        focus() {
+            const ele = this.$refs.input || this.$refs.textarea;
```
Comment (JosephusPaye) [inline src/UiTextbox.vue]: You can avoid using a temporary variable like so:

```js
(this.$refs.input || this.$refs.textarea).focus();
```

**Comment by JosephusPaye** [comment]
Hi @myerscw,

Good work, see my comment on the file.

**Comment by harry75369** [comment]
Just curious, why is this feature still not merged or implemented yet? We need focus, right?

**Comment by JosephusPaye** [comment]
Merged, thanks!

**Pull request merged**